plugins/qmlsupport/CMakeLists.txt

```
@@ -11,16 +11,16 @@ set(gammaray_qmlsupport_srcs
   qmltypeextension.cpp
 )
 
+if(NOT Qt5Qml_VERSION VERSION_LESS 5.10)
+    set(gammaray_qmlsupport_srcs ${gammaray_qmlsupport_srcs} qmlbindingprovider.cpp)
```

Alternative:
```suggestion
    list(APPEND gammaray_qmlsupport_srcs qmlbindingprovider.cpp)
```
...I don't really have any insights into whether there's a performance difference, or which form would come out ahead if there is. I just know that `list(APPEND...` is available in any CMake 3.0+ (at least), and it avoids one repetition of the variable name.